InvenTree/part/test_api.py

```diff
@@ -293,6 +294,113 @@ def test_invisible_chars(self):
 
             self.assertEqual(response.data['description'], 'A part category')
 
+    def test_category_delete(self):
+        """Test category deletion with different parameters"""
+
+        class Target(IntEnum):
+            move_subcategories_to_parent_move_parts_to_parent = 0,
+            move_subcategories_to_parent_delete_parts = 1,
+            delete_subcategories_move_parts_to_parent = 2,
+            delete_subcategories_delete_parts = 3,
+
+        for i in range(4):
```

Comment:
Question(non-blocking): Would a smaller range also be sufficient to get full coverage? Our tests are already running quite long

Reply:
Is there any way to get a coverage info from the test runs?
At the moment I as I see we can improve it (one case for part delete, one case for category delete and a third for move to parent for both), but in the case in the future the code changes it might loose coverage.

Reply:
@martonmiklos you can look into the check runs. There is a check from coveralls with the coverage and a link.  Alternatively, you can check the SQLite + coverage check. there is a step for uploading coverage, in the logs, you will find a link to the coverage results.

I think one case less would be sufficient to cover the changes but let's leave the tests as is. In the future, a more explanatory docstring about the test function - especially with such a clever duplication reduction mechanism - would probably speed up reviewing.